redesign/generators.py

```diff
@@ -1,44 +1,55 @@
 """Contains generators for tabular, graph, and unstructured data profiles."""
 
+from synthetic_data.synthetic_data import make_data_from_report
+
 from .base_generator import BaseGenerator
 
 
 class TabularGenerator(BaseGenerator):
     """Class for generating synthetic tabular data."""
 
-    def __init__(self, profile, options):
+    def __init__(self, profile, seed=None, noise_level: float = 0.0):
         """Initialize tabular generator object."""
-        super().__init__(profile, options)
+        super().__init__(profile, seed)
+        self.noise_level = noise_level
 
-    def synthesize(self):
+    def synthesize(self, num_samples: int, seed=None, noise_level: float = None):
```

Comment:
should `seed` still be here? it is also in the `__init__`...

Reply:
The user can pass in a different seed than the one in `__init__` if they want. Not sure if we need this. @JGSweets

Reply:
the intent would be to override if the user wants to change it for that given generation